### DISCUSSION TITLE
In 167.1 - large-v3 now is using the official model.

### DISCUSSION THREAD

**Discussion by 1001ruchka** [Purfview/whisper-standalone-win#150] [Q&A] [answered]
In 167.1 - large-v3 now is using the official model.

What does it mean?

You answered:

> Logically it should mean that before it was using a non-official model, no?

I am using Subtitle Edit and the model that this program downloaded. How can I download the official version of the model?
I thought the file format of the fast-whisper model was different from whisper.cpp

https://huggingface.co/Systran/faster-whisper-large-v3
Is this version the official version?

And why did versions from 160.x become 167.x? Has there been a jump forward a few versions?

**Comment by Purfview** [comment, marked as answer]
> I am using Subtitle Edit and the model that this program downloaded. How can I download the official version of the model?

Current Subtitle Edit version auto-downloads `r160.7`,  that version is using non-official model because it provided the v3 model a month before it was officially released.

If you use it in terminal then it audownloads models, for SE you can download them from there -> https://huggingface.co/Systran

> I thought the file format of the fast-whisper model was different from whisper.cpp.

Yes, it's different.


> And why did versions from 160.x become 167.x? Has there been a jump forward a few versions?

Major version number is the Commits/revisions included from the original repo:

![Screenshot_1](https://github.com/Purfview/whisper-standalone-win/assets/69023953/fa6fe14c-daa0-4d5e-a9cc-02dd0dcc2227)

**Reply by ghost** [reply]
ahhhh subtitle edit okay.

**Reply by Purfview** [reply]
> What is SE, which one do you recommend? that auto downloaded or SE?

"SE" is short for [Subtitle Edit](https://github.com/SubtitleEdit/subtitleedit), it can be used as a GUI.

Can't answer the other questions as they don't make sense.
